frontend-html/src/gui/Workbench/ScreenArea/FormView/FormViewEditor.tsx

```diff
@@ -46,9 +46,18 @@ import { runInFlowWithHandler } from "utils/runInFlowWithHandler";
 import ColorEditor from "gui/Components/ScreenElements/Editors/ColorEditor";
 import { CellAlignment } from "gui/Components/ScreenElements/Table/TableRendering/cells/cellAlignment";
 import { flashColor2htmlColor, htmlColor2FlashColor } from "@origam/utils";
-import { isAddRecordShortcut } from "utils/keyShortcuts";
+import {
+  isAddRecordShortcut,
+  isDeleteRecordShortcut,
+  isDuplicateRecordShortcut, isFilterRecordShortcut,
```

Comment:
You import but not handle the isFilterRecordShortcut?

Reply:
Are you suggesting it should be handled? I think the import isFilterRecordShortcut should be removed.